src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/osd/osd-pg-scrub-modal/osd-pg-scrub-modal.component.html

```diff
@@ -0,0 +1,49 @@
+<cd-modal [modalRef]="bsModalRef">
+  <ng-container i18n="form title|Example: Create Pool@@formTitle"
+                class="modal-title">{{ action | titlecase }} {{ resource | upperFirst }}</ng-container>
+
+  <ng-container class="modal-content">
+    <form class="form-horizontal"
+          #formDir="ngForm"
+          [formGroup]="osdPgScrubForm"
+          novalidate>
+      <div class="modal-body osd-modal">
+        <!-- Basic -->
+        <cd-config-option [optionNames]="basicOptions"
+                          [optionsForm]="osdPgScrubForm"
+                          [optionsFormDir]="formDir"
+                          [optionsFormGroupName]="'basicFormGroup'"
+                          #basicOptionsValues></cd-config-option>
+        <!-- Advanced -->
+        <div class="row">
+          <div class="col-sm-12">
+            <a class="pull-right margin-right-md"
+               (click)="advancedEnabled = true"
+               *ngIf="!advancedEnabled"
+               i18n>Advanced...</a>
+          </div>
+        </div>
+        <div *ngIf="advancedEnabled">
+          <h2 class="page-header"
+              i18n>Advanced configurations</h2>
```

Comment:
"Advanced configurations" sounds wrong to me. How about this?

```suggestion
              i18n>Advanced configuration options</h2>
```